README.rst

```diff
@@ -27,6 +27,16 @@ Look at the source of ``templates/admin/base_site.html`` and override the templa
 For example, you can add your own CSS in ``templates/admin/inc/extrastyle.html``.
 
 
+Settings
+========
+
+* ``CMS_ENABLE_UPDATE_CHECK = True``
+  Set to ``False`` to disable the update notification.
+* ``CMS_UPDATE_CHECK_TYPE = ('patch')``
```

Comment:
This is not right.
Can't be set to both, it's either `patch` or `minor`.
Currently it defaults to `minor`.